src/index.html

```diff
@@ -4,15 +4,89 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     />
     <title>Stars</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./style-stars--0.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style-stars--1.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style-stars--2.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style-stars--3.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style-stars--4.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style-stars--5.css"
+    />
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+    </ul>
```

Comment:
Add an empty line between sibling blocks of HTML to add some 'air' and simplify reading.